config/initializers/excon_instrumentation.rb

```diff
@@ -0,0 +1,6 @@
+require 'pvb/excon/instrument_factory'
+
+ActiveSupport::Notifications.subscribe(/excon/) do |event, start, finish, _id, payload|
+  instrument = PVB::Excon::InstrumentFactory.for(event, start.to_f, finish.to_f, payload)
```

Comment:
Can you add a comment roughly explaining how Excon instrumentation works? ie when a request happens it calls the generates a 'request', 'response' events, etc ? I think it will make it easier to maintain down the line having a bit of an intro.